src/PJ_rpoly.c

```diff
@@ -63,7 +96,8 @@ PJ *PROJECTION(rpoly) {
     }
     P->es = 0.;
     P->fwd = s_forward;
-
+    P->inv = s_inverse;
+    
     return P;
 }
 
```

Comment:
Sure, I'll use their data for the self-tests. I'm already using it in ad-hoc `cs2cs` scripts for testing.